### PR TITLE
fix: add more logs and fetch fresh google creds each attestation

### PR DESCRIPTION
**Pull request by bryce-mcmath**
Adding more logs and moving the google credentials from global scope to function scope, fetched every time so they don't expire over time. I'm hoping that's what was causing the problems after a day or so.

### CODE REVIEW

File: src/goog.py [hunk 1]
```diff
@@ -54,6 +49,10 @@ def isValidVerdict(verdict, nonce):
 # decrypt the integrity token on google's servers
 def verify_integrity_token(token, nonce):
     try:
+        path = os.getenv("GOOGLE_AUTH_JSON_PATH")
+        creds = service_account.Credentials.from_service_account_file(
+            path, scopes=["https://www.googleapis.com/auth/playintegrity"]
```
Comment (jleach) [inline src/goog.py:54]: constants.py?

File: src/traction.py [hunk 2]
```diff
@@ -35,9 +36,13 @@ def fetch_bearer_token():
         response_data = json.loads(response.text)
 
         bearer_token = response_data["token"]
+        if bearer_token is None:
+            logger.info("Token doesn't exist in response data")
+
         return bearer_token
     else:
         logger.info(f"Error fetcing token: {response.status_code}")
+        logger.info(f"Text content for error: {response.text}")
```
Comment (jleach) [inline src/traction.py:45]: Do we have "logger.error"?

**Review by jleach — CHANGES_REQUESTED** [review]

File: src/goog.py (hunk 1 above)
Comment (bryce-mcmath) [reply src/goog.py]: Done

File: src/traction.py (hunk 2 above)
Comment (bryce-mcmath) [reply src/traction.py]: Yes, done

**Review by jleach — APPROVED** [review]

**Pull request merged**